src/orders/components/OrderManualTransactionRefundPage/components/OrderManualTransactionRefundTiles/OrderManualTransactionRefundTiles.tsx

```diff
@@ -52,15 +53,27 @@ export const OrderManualTransationRefundTiles = ({
                           disabled={!isRefundable}
                           error={!!error}
                           padding={4}
+                          paddingTop={0}
+                          width="100%"
+                          display="grid"
+                          __gridTemplateColumns="max-content 1fr"
                         >
-                          <Text
-                            size={5}
-                            fontWeight="medium"
-                            padding={4}
-                            color={isRefundable ? "default1" : "defaultDisabled"}
+                          <Box
+                            display="flex"
+                            justifyContent="space-between"
+                            alignItems="center"
+                            width="100%"
                           >
-                            {transaction?.name || "Transaction"}
-                          </Text>
+                            <Text
+                              size={5}
+                              fontWeight="medium"
+                              padding={4}
+                              color={isRefundable ? "default1" : "defaultDisabled"}
+                            >
+                              {transaction?.name || "Transaction"}
```

Comment:
question: `"Transaction"` should't be translated?

Reply:
It was like that before but you're right I will add it to translations